src/elm/Page/Transfer.elm

```diff
@@ -0,0 +1,414 @@
+                        ]
+                    ]
+                , div [ class "bg-no-repeat bg-auto h-64 ml-32 -mt-5 px-4 py-2 m-2 items-center mt-5 justify-center", style "background-image" "url(/images/transfer-doggo.svg)" ]
+                    []
+                , div [ class "flex sm:justify-center sm:items-center -mt-16 absolute md:w-full md:mx-auto  sm:right-0" ]
+                    [ viewTransferCard loggedIn transfer state
+                    ]
+                ]
+            ]
+        ]
+
+
+viewTransferCard : LoggedIn.Model -> Transfer -> State -> Html Msg
+viewTransferCard loggedIn transfer state =
+    let
+        ipfsUrl =
+            loggedIn.shared.endpoints.ipfs
+    in
+    div [ class "flex flex-row block rounded overflow-x-scroll sm:overflow-hidden bg-gray-100" ]
+        [ div [ class "sm:px-4 sm:py-2 sm:m-2 p-0" ]
+            [ div [ class "h-8 w-8 rounded-full mx-auto mt-5" ]
+                [ Avatar.view ipfsUrl
+                    (case state of
+                        Received ->
+                            transfer.to.avatar
+
+                        Transferred ->
+                            transfer.from.avatar
+
+                        NotInvolved ->
+                            transfer.from.avatar
+                    )
+                    "h-10 w-10"
+                ]
+            , div [ class "px-6 py-4" ]
+                [ span [ class "text-base inline-block bg-black rounded-full px-3 py-1 text-sm font-semibold text-white" ]
+                    [ case state of
+                        Received ->
+                            text "You"
+
+                        Transferred ->
+                            text "You"
+
+                        NotInvolved ->
+                            Eos.viewName transfer.fromId
+                    ]
+                ]
+            ]
+        , div [ class "mt-5" ] [ viewAmount loggedIn transfer state ]
+        , div [ class "-ml-20 sm:px-4 sm:py-2 m-2" ]
+            [ div [ class "h-8 w-8 rounded-full mx-auto mt-5" ]
+                [ Avatar.view ipfsUrl
+                    (case state of
+                        Received ->
+                            transfer.from.avatar
+
+                        Transferred ->
+                            transfer.to.avatar
+
+                        NotInvolved ->
+                            transfer.to.avatar
+                    )
+                    "h-10 w-10"
+                ]
+            , div [ class "px-6 py-4" ]
+                [ span [ class "inline-block bg-black rounded-full px-3 py-1 text-sm font-semibold text-white" ]
+                    [ Eos.viewName <|
+                        case state of
+                            Received ->
+                                transfer.fromId
+
+                            Transferred ->
+                                transfer.toId
+
+                            NotInvolved ->
+                                transfer.toId
+                    ]
```

Comment:
@Pancakem actually we need to try to display the user name, and if not present or `""` we should then display the account. 

You'll have to modify the transfer query to also search for the user's name. and do a logic to display account or user's name depending on the contents of the name

Reply:
For example, my username is `julienlucca` but my name is `Julien Lucca`. We should display the later since on my profile we have that information